pulsar-broker/src/test/java/org/apache/pulsar/broker/web/WebServiceTest.java

```diff
@@ -349,6 +349,20 @@ public void testBrokerReady() throws Exception {
         assertEquals(res.getResponseBody(), "ok");
     }
 
+    @Test
+    public void testCompressMetricsData() throws Exception {
```

Comment:
Is it able to check whether the data has been compressed? I think the test can also get passed if the flag is disabled.